app/services/fuzzySearchService.js

```diff
@@ -102,6 +102,13 @@ const groupByPayee = (searchResults) => {
   return result
 }
 
+const getSearchSuggestions = async (searchKey) => {
+  const paymentData = await getAllPaymentData()
+  const fuse = new Fuse(paymentData, fuseSearchOptions)
+  const searchResult = fuse.search(searchKey).map(row => row.item)
+  return searchResult.slice(0, 6)
```

Comment:
Can we define the magic number 6 somewhere as the search suggestion results limit?

Reply:
agreed, and moved to appconfig file now.

Reply:
I think we can simplify searchSuggestionResultsLimit if we put it within the search object in appConfig and the name can also be shortened to suggestionResultsLimit as we know its the search config.

Reply:
ok, moved the variable & formatted the name now.